quickjs.c

```diff
@@ -30306,6 +30549,12 @@ static JSValue js_create_function(JSContext *ctx, JSFunctionDef *fd)
     b->arguments_allowed = fd->arguments_allowed;
     b->backtrace_barrier = fd->backtrace_barrier;
     b->realm = JS_DupContext(ctx);
+    b->ic = fd->ic;
```

Comment:
```suggestion
    b->ic = fd->ic;
    fd->ic = NULL;
```
Looks like a potential double-free otherwise.